code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java

```diff
@@ -1,6 +1,8 @@
-package ru.unn.agile.ComplexNumber.model;
+package ru.unn.agile.ComplexNumber.Model;
 
```

Comment:
You can't change any files in this repo except your own, please pay attention.

Reply:
Gradle can't be built without that change